Quorum/apis/llms/llm_conversation.py

```diff
@@ -0,0 +1,105 @@
+from enum import StrEnum, auto
+from pathlib import Path
+
+class MessageRole(StrEnum):
+    USER = auto()
+    ASSISTANT = auto()
+
+
+MAX_TOKENS = 1024
+
+
+def tmp_send_message_to_claude(client, model, messages):
```

Comment:
Make sure typing in call variables and return value. applicable for all changes in this PR.

Reply:
This whole function is temporary and I will probably remove it before merging this PR. I made sure to make it as ugly as possible for this reason. This is just because @yoav-el-certora asked me to notice how many API calls I make during development so we estimate costs and I didn't want to write manually every time I call the LLM.